### PR TITLE
added random function

### LINKED ISSUE #174: Random function
Thanks for your great work!

Can you add Random builtin function support, please?

### PR DESCRIPTION
**Pull request by vcolluto**
Added a random function in run.js. The function returns a random number between 0 and the value of the parameter (excluding)

Fixes #174

### CODE REVIEW

File: js/run.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
  # *************************************************************************************
  */
 
-let strokeLow = {
+ let strokeLow = {
```
Comment (rajkumaar23) [inline js/run.js:22]:
```suggestion
let strokeLow = {
```

File: js/run.js [hunk 2]
```diff
@@ -1374,4 +1374,8 @@ function mathFunctions(variableName, userInput) {
             return globalEval(variableName + '=' + 'Math.round(' + a + ')')
         }
     }
+    if (userInput.includes('random')) {
+        let variable = parseFloat(globalEval(parametersAsString[1]))
+        return globalEval(variableName + '=' + 'Math.floor(Math.random() *' + variable + ')')
```
Comment (rajkumaar23) [inline js/run.js:1379]:
```suggestion
        return globalEval(variableName + '=' + 'Math.round(Math.random() *' + variable + ')')
```

I believe rounding is a better choice than floor-ing. 🤔

**Review by rajkumaar23 — CHANGES_REQUESTED** [review]
The fn works, just one small nit. 

Also, can you please make another PR [here](https://github.com/flowgrammers-org/flowgramming-manual/blob/main/src/math-functions.md) to update the help docs?

File: js/run.js (hunk 1 above)
Comment (vcolluto) [reply js/run.js]: done!

**Review by rajkumaar23 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**